### ISSUE TITLE
Encryption at rest

### ISSUE THREAD

**Issue by nkosi23** [meganz/android#76]
Is the Android client storing account data in plain text on the device?

**Comment by mega-ht** [comment]
No, we encrypt the sensitive data before storing locally in the device.
